### ISSUE TITLE
Change `base_url` in configuration to any value causes `ErrorException`

### ISSUE THREAD

**Issue by mvolpato** [statamic/ssg#39]
I am trying several options as `base_url` in the configuration file `config/statamic/ssg.php`, but the only accepted one is the default one `app.url`. This is very weird, and I have no idea how this is possible.

Output:

```
   ErrorException 

  Undefined offset: 1

  at vendor/statamic/ssg/src/Request.php:33
     29▕     }
     30▕ 
     31▕     public function getHttpHost()
     32▕     {
  ➜  33▕         $withoutScheme = rtrim(explode('://', $this->config['base_url'])[1], '/');
     34▕ 
     35▕         $length = strlen($this->getBaseUrl());
     36▕ 
     37▕         return ($length === 0) ? $withoutScheme : substr($withoutScheme, 0, -$length);

      +8 vendor frames 
  9   [internal]:0
      Illuminate\Support\Collection::Illuminate\Support\Traits\{closure}(Object(Statamic\StaticSite\Page))

      +21 vendor frames 
  31  please:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

Tested: `app.com`, `test.url`, `https://app.url`, `http://app.url`, `https://app.url/`, `google.com`, `https://google.com`.

Version: 0.2.0

**Comment by jasonvarga** [comment]
Use one with http:// or https:// in it, run php artisan confit:clear, and try it again.

**Comment by mvolpato** [comment]
Tried `php artisan config:clear` and `php artisan cache:clear` but I still get the same error.

Looking into `php artisan` now if there is some command I was missing. (not a php dev).

**Comment by jasonvarga** [comment]
I can't reproduce this. It seems the same as #1 (which is fixed in #51) but goes away if you use a url with http:// or https:// in it as I explained above. Feel free to reopen with more details.
